wurst/objects/abilities/beastmaster/Druid/BarkSkin.wurst

```diff
@@ -0,0 +1,89 @@
+package BarkSkin
+
+// Standard Library Imports:
+import Icons
+import ChannelAbilityPreset
+import ClosureEvents
+import ClosureTimers
```

Comment:
ClosureEvents & CloresureTimers are unused? If yes, remove them.

Reply:
Ahh those were left in from the first implementation.